models/staging/edfi_3/stage/stg_ef3__stu_ed_org__languages.sql

```diff
@@ -0,0 +1,45 @@
+with stg_stu_ed_org as (
+    select * from {{ ref('stg_ef3__student_education_organization_associations') }}
+),
+stg_descriptors as (
```

Comment:
remove this object -- we try to avoid joins in staging models, and can handle this downstream